fs/io_zenfs.cc

```diff
@@ -83,6 +86,11 @@ Status ZoneFile::DecodeFrom(Slice* input) {
   if (tag != kFileID || !GetFixed64(input, &file_id_))
     return Status::Corruption("ZoneFile", "File ID missing");
 
+  // Zonefiles created by a previous zenfs version may not have the
+  // modification time stamp. Show epoch time stamp if it is not set
+  // previously.
+  m_time_ = 0;
```

Comment:
Good comment, but move m_time_ to the constructor

Reply:
Yes, that is a better idea. Now the constructor initializes m_time_ to 0, on file creation, m_time_ is also updated, which will be overwritten on subsequent syncs.